react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+export default class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      imgList: props.images,
```

Comment:
Generally not a good idea to set `props` to `state`. If the `props` change the `state` won't reflect the change so the component won't update as expected.